modules/web-console/e2e/testcafe/components/notifications.js

```diff
@@ -18,3 +18,4 @@ import {Selector} from 'testcafe';
 
 export const successNotification = Selector('body > .alert-success');
 export const errorNotification = Selector('body > .alert-danger');
+export const popoverErrorNotification = Selector('.popover.validation-error');
```

Comment:
Please move this to sql configuration page model. No other pages use this element. I think you got me wrong.

Reply:
Moved.